src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -678,6 +678,21 @@ - (BOOL)deviceBeingCommissionedOverBLE:(uint64_t)deviceId
     return deviceProxy->GetDeviceTransportType() == chip::Transport::Type::kBle;
 }
 
+- (NSData *)convertX509CertToMatterCert:(NSData *)x509Cert
```

Comment:
It's difficult to evaluate the usage of this without any callers in the PR.  Is the need for this a symptom of the fact that the OperationalCredentialsDelegate interface outputs certificates in DER format?

Reply:
> It's difficult to evaluate the usage of this without any callers in the PR. Is the need for this a symptom of the fact that the OperationalCredentialsDelegate interface outputs certificates in DER format?

@msandstedt : please check below commit on Android. We want to use same style on different platform(iOS/Android) for unified style of our application. so we want to add this. please check again it.
- https://github.com/project-chip/connectedhomeip/pull/15551